### ISSUE TITLE
Selecting multiple images when only clicking on one.

### ISSUE THREAD

**Issue by Manu-Devloo** [immich-app/immich#8216]
### The bug

When i go to select a image in my library it will select the one I want but then shift up slightly and there for also selecting the images between the one I wanted to select and the one underneath.

### The OS that Immich Server is running on

Unraid 

### Version of Immich Server

1.99

### Version of Immich Mobile App

1.99

### Platform with the issue

- [ ] Server
- [ ] Web
- [x] Mobile

### Your docker-compose.yml content

```YAML
version: "3.8"

#
# WARNING: Make sure to use the docker-compose.yml of the current release:
#
# https://github.com/immich-app/immich/releases/latest/download/docker-compose.yml
#
# The compose file on main may not be compatible with the latest release.
#

name: immich

services:
  backup:
      container_name: immich_db_dumper
      image: prodrigestivill/postgres-backup-local
      env_file:
        - .env
      environment:
        POSTGRES_HOST: database
        POSTGRES_DB: ${DB_DATABASE_NAME}
        POSTGRES_USER: ${DB_USERNAME}
        POSTGRES_PASSWORD: ${DB_PASSWORD}
        SCHEDULE: "@daily"
        BACKUP_DIR: /db_dumps
      volumes:
        - ${BACKUP}:/db_dumps
      depends_on:
        - database
  immich-server:
    container_name: immich_server
    image: ghcr.io/immich-app/immich-server:${IMMICH_VERSION:-release}
    command: [ "start.sh", "immich" ]
    volumes:
      - ${UPLOAD_LOCATION}:/usr/src/app/upload
      - /etc/localtime:/etc/localtime:ro
    env_file:
      - .env
    ports:
      - 2283:3001
    depends_on:
      - redis
      - database
    restart: always

  immich-microservices:
    container_name: immich_microservices
    image: ghcr.io/immich-app/immich-server:${IMMICH_VERSION:-release}
    # extends: # uncomment this section for hardware acceleration - see https://immich.app/docs/features/hardware-transcoding
    #   file: hwaccel.transcoding.yml 
    #   service: cpu # set to one of [nvenc, quicksync, rkmpp, vaapi, vaapi-wsl] for accelerated transcoding
    command: [ "start.sh", "microservices" ]
    volumes:
      - ${UPLOAD_LOCATION}:/usr/src/app/upload
      - /etc/localtime:/etc/localtime:ro
    env_file:
      - .env
    depends_on:
      - redis
      - database
    restart: always

  immich-machine-learning:
    container_name: immich_machine_learning
    # For hardware acceleration, add one of -[armnn, cuda, openvino] to the image tag.
    # Example tag: ${IMMICH_VERSION:-release}-cuda
    image: ghcr.io/immich-app/immich-machine-learning:${IMMICH_VERSION:-release}
    # extends: # uncomment this section for hardware acceleration - see https://immich.app/docs/features/ml-hardware-acceleration
    #   file: hwaccel.ml.yml
    #   service: cpu # set to one of [armnn, cuda, openvino, openvino-wsl] for accelerated inference - use the `-wsl` version for WSL2 where applicable
    volumes:
      - model-cache:/cache
    env_file:
      - .env
    restart: always

  redis:
    container_name: immich_redis
    image: redis:6.2-alpine@sha256:afb290a0a0d0b2bd7537b62ebff1eb84d045c757c1c31ca2ca48c79536c0de82
    restart: always

  database:
    container_name: immich_postgres
    image: tensorchord/pgvecto-rs:pg14-v0.2.0@sha256:90724186f0a3517cf6914295b5ab410db9ce23190a2d9d0b9dd6463e3fa298f0
    env_file:
      - .env
    environment:
      POSTGRES_PASSWORD: ${DB_PASSWORD}
      POSTGRES_USER: ${DB_USERNAME}
      POSTGRES_DB: ${DB_DATABASE_NAME}
    volumes:
      - ${DATABASE}:/var/lib/postgresql/data
    restart: always

volumes:
  pgdata:
  model-cache:
```

### Your .env content

```Shell
# You can find documentation for all the supported env variables at https://immich.app/docs/install/environment-variables

# The location where your uploaded files are stored
UPLOAD_LOCATION=/mnt/user/images/immich
DATABASE=/mnt/user/images/immich/dataBase
BACKUP=/mnt/user/Backups/immich/database

# The Immich version to use. You can pin this to a specific version like "v1.71.0"
IMMICH_VERSION=release

# Connection secrets for postgres and typesense. You should change these to random passwords
DB_PASSWORD=postgres

# The values below this line do not need to be changed
###################################################################################
DB_HOSTNAME=immich_postgres
DB_USERNAME=postgres
DB_DATABASE_NAME=immich

REDIS_HOSTNAME=immich_redis
```

### Reproduction steps

```bash
1. Open immich
2. Try to select an item, hold in the bottom half.
```

**Comment by nando2301** [comment]
Hi, please add Select All button in Web and Mobile app, when you try to create another album with a lot of images in People Álbum is so bad select one by one, Thanks

**Comment by JordyEGNL** [comment]
I am having the same problem, when long pressing on an image to select that image it will shift the whole screen up and it will select more images unexpectedly

**Comment by JordyEGNL** [comment]
This has been fixed in #9026
